modules/configuring-network-baselining-timeframe.adoc

```diff
@@ -9,19 +9,20 @@ You can use the `ROX_NETWORK_BASELINE_OBSERVATION_PERIOD` and the `ROX_BASELINE_
 
 .Procedure
 
-* Set the `ROX_NETWORK_BASELINE_OBSERVATION_PERIOD` and the `ROX_BASELINE_GENERATION_DURATION` environment variables:
+. Set the `ROX_NETWORK_BASELINE_OBSERVATION_PERIOD` environment variable:
 +
 [source,terminal]
 ----
-$ oc -n stackrox set env deploy/central \ <1>
+$ oc -n stackrox set env deploy/central \ //<1>
```

Comment:
hmm...jw is the // supposed to comment out this callout? Maybe a # would do the trick?

Reply:
Oops, I was following guidance given during a previous peer review, but I should have left out the space:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks